packages/frontend/app/pods/components/pod/chat/chat-entry/emoji-parser.ts

```diff
@@ -0,0 +1,23 @@
+const emojiDict = new Map([
+    [":partyparrot:", "'(\\(0>0)/)'"] // TODO: use gif
+]);
+
+export default class EmojiParser {
```

Comment:
because this class doesn't maintain any state, it could probably be converted into a pure function